qdrant-landing/themes/qdrant-2024/assets/css/partials/_qdrant-pricing-doors-b.scss

```
@@ -0,0 +1,33 @@
+@use '../helpers/functions' as *;
+
+.qdrant-pricing-doors-b {
+  padding-bottom: $spacer * 4;
+  padding-top: $spacer * 4;
+  background-color: $neutral-10;
+
+  &__doors {
+    display: flex;
+    flex-direction: column;
+    gap: rem(30);
+    width: 100%;
+  }
+
+  @include media-breakpoint-up(xl) {
+    padding-bottom: $spacer * 5;
+    padding-top: $spacer * 5;
+
+    &__doors {
+      flex-direction: row;
+      justify-content: space-between;
```

```suggestion
      justify-content: center;
```

**See an explanation in the next comment**